themes/default/content/blog/new-improved-pulumi-service-console/index.md

```diff
@@ -0,0 +1,89 @@
+tags:
+    - pulumi-service
+    - user-experience
+
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md.
+# for additional details, and please remove these comments before submitting for review.
+---
+
+It’s been nearly four years since the Pulumi Service console launched back in 2018. The [Pulumi Service]({{< relref "/product/pulumi-service" >}}) is the easiest way to use Pulumi’s open source, universal infrastructure as code SDK at scale and provides a fully managed experience. It handles infrastructure state and secrets, sets up SAML SSO, integrates with CI/CD pipelines, and enforces compliance rules.
+
+In these last four years, our customer base has grown significantly Service’s features and capabilities have grown quickly. We’ve added features like [audit logs]({{< relref "/docs/intro/pulumi-service/audit-logs" >}}), [SAML SSO]({{< relref "/docs/guides/saml" >}}) and [SCIM]({{< relref "/docs/guides/scim" >}}), [Teams]({{< relref "/docs/intro/pulumi-service/teams" >}}), [Organization]({{< relref "/docs/intro/pulumi-service/organization-access-tokens" >}}) and [Team Access Tokens]({{< relref "/docs/intro/pulumi-service/team-access-tokens" >}}), and much more! But throughout all these changes, the look and feel of the Service has largely stayed the same—until now!
+
+<!--more-->
+Today we’re excited to launch a set of improvements to the overall user experience of the Pulumi Service. These updates include:
+
+- Increased organizational insights on the Service Dashboard
+- Quick access to the stacks you care about most with a new Favoriting feature
+- Easier navigation around the Service
+
+These changes came directly from consultation with our users and we've spent time with customers in the past few weeks refining it. One of these customers was [dutchie](https://dutchie.com), an all-in-one technology platform powering the cannabis industry with Point of Sale, Ecommerce, Payments, and more. "The new Pulumi Service UI is a change we are really excited to see, literally. It has better use of white space and an improved balance of information on the Dashboard at the same time and that shows a real and welcome focus on user experience. It is a genuine quality of life improvement in a portion of the product that we use daily." said Jordan Rinke, Infrastructure Architect at dutchie.
+
+Want to try these features out for yourself? Next time you sign in to your Pulumi Service account, select the toggle in the black utility toolbar at the top of the page to “Try the new Pulumi Service!” You can take these new changes for a ride, or return to the old experience at any time.
+
+## Easily navigate across the Pulumi Service with the new left navigation pane
+
+Among the biggest changes to the feel of the Service is our new left navigation pane. Navigation items like your Dashboard, Stacks, and Settings are accessible in the new pane from anywhere in the Service.
+
+![""](dashboard-nav-expanded.png)
```

Comment:
Include alt text?

Reply:
we can include alt text if you would like. screenshots are especially difficult to write good alt text for especially when they are dense in information. if the content contained within the screenshot is described on the page it is not needed.